### PR TITLE
Init Polish localization

### PR DESCRIPTION
**Pull request by mfilocha**
An initial config for Polish localization.

### CODE REVIEW

**Comment by sftim** [comment]
Am I right that this is a work-in-progress?

**Comment by sftim** [comment]
@mfilocha would you like help with, for example, getting a new branch made for the Polish work?

**Comment by mfilocha** [comment]
@sftim - @jimangel and @rlenferink asked me to reopen my previous PR against the master branch: https://github.com/kubernetes/website/pull/18254#issuecomment-569769916

**Comment by rlenferink** [comment]
@sftim, we want to have this merged to master (our latest) branch as described by Jim as well. Why do you want to add `pl` to the `disableLanguages`? This adds all the initial configs and it will be a lot easier to translate and review when this will be deployed for every PR.

As I understand @mfilocha processed all feedback from the previous pull request and created this one after processing the feedback. The only thing left to do is for @nvtkaszpir to request [org membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) (because OWNERS files are checked automatically) (@remyleone and me offered to sponsor).

**Comment by mfilocha** [comment]
@remyleone

**Comment by sftim** [comment]
This pull request does not yet add the [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content) for a new localization. For that reason, it's not ready for approval.

According to the contribution guide, this new localization needs:
- A translation of the CNCF code of conduct into Polish
- A home page with all heading and subheading URLs
- A setup section - https://kubernetes.io/pl/docs/setup/ - with heading and subheading URLs 
- The _Kubernetes basics_ tutorial
- The _Hello Minikube_ tutorial
- All site strings in a new localized TOML file

That's quite some work. So I'm recommending starting a new branch and working on making a series of changes to the new branch, until it is ready to merge into _master_.

**Comment by sftim** [comment]
@kubernetes/sig-docs-l10n-admins can you advise on whether I've [mis]understood?

**Comment by mfilocha** [comment]
I'll follow whatever is the recommended way. 
My biggest concern is that the PR containing all translated minimum will be of XXL size. And who will review this PR for correctness/style of translation?

**Comment by sftim** [comment]
@mfilocha you're talking about the merge from the Polish localization dev branch into _master_? Because that XXL merge will consist only of already-approved changes, it won't need code review, just signoff.

For an example, see commit 487505d03883a73a00fa436e195b66a67d9090aa from PR #16551. I'm assuming that this single commit is the result of squashing a branch that includes many PRs (I haven't actually done one of these; that's what it looks like to me).

**Comment by remyleone** [comment]
I think targetting master is the way to go to start a localization and can be proven by the following successfully merged init localization: https://github.com/kubernetes/website/pull/12548 https://github.com/kubernetes/website/pull/16965/files

**Comment by remyleone** [comment]
@mfilocha got already a lot of content translated in Polish in his own fork ( https://github.com/mfilocha/website/tree/dev-1.16-pl.1 ). 
My take would be to merge this initial PR as quickly as possible to make @mfilocha and the rest of the Polish translation team as autonomous as possible. Having this PR merged will make them able to quickly merge a lot of already made content that will be beneficial to the Polish-speaking users.

Slowing it down with extra requirements (that were not enforced for instance in the past of the Vietnamese translation) is not justified in this case IMHO. 

@nvtkaszpir let me know when you open your org membership request, I will give my sponsoring to it.

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/17388

I agree: the current guidelines aren't clear, but there's a bug open to fix that.

**Comment by sftim** [comment]
@remyleone can I follow up on https://github.com/kubernetes/website/pull/18419#issuecomment-570273563 ?

#12548 looks like an example of a valid new localization, whereas it looks to me as if PR #16965 was merged too early (it should have waited on the changes that came in #17655). The Vietnamese localization team are doing a great job though, they've landed _lots_ of good PRs since that first one.

@mfilocha I do not think that https://deploy-preview-18419--kubernetes-io-master-staging.netlify.com/pl/ looks good to merge; to me, this pull request looks like a strong work-in-progress. You can `/retitle [WIP] Init Polish localization` if you agree that there's more to do.

**Comment by mfilocha** [comment]
I wonder if dropping [config.toml](https://github.com/kubernetes/website/pull/18419/files#diff-618063036395fe9ee107f22b46c9eade) from this PR could help to avoid the scenario with the empty page as pointed by @sftim? 
If I'm right this may allow us to follow the way set out by @remyleone  in https://github.com/kubernetes/website/pull/18419#issuecomment-570276761.

**Comment by remyleone** [comment]
@sftim I disagree and think that a team should be able to show results as fast as possible and I'm not in favor of enforcing an arbitrary number of PR before they can have the gratification to see their work live. Wikipedia is a clear demonstration of this principle. Plus as you trust/empower a team as early as possible they have all agency to show other potential contributors that they can to quickly have an impact. All actions that delay the gratification of seeing your work in production should be removed in IMHO

**Comment by mfilocha** [comment]
PR to create a label for Polish language https://github.com/kubernetes/test-infra/pull/15684 is on hold waiting for this PR to be merged.

**Comment by sftim** [comment]
To make https://deploy-preview-18419--kubernetes-io-master-staging.netlify.com/pl/ look right you need at least an `_index.html` page and to localize all the strings it uses.

The way I see it, the minimum guidelines do indeed cover what's necessary to get a sensible-looking preview rendering OK for a new localization.

There are 2 “essential” pages in the minimum requirements; one is _Hello Minikube_ and the other is the _Kubernetes basics_ tutorial. I can see why those are included, they are really useful for people new to Kubernetes. It's OK to launch a localization once those 2 pages are ready, even before the remaining hundreds of pages are ready to publish.

**Comment by mfilocha** [comment]
@sftim All required translated content is ready at my branch: https://github.com/kubernetes/website/compare/master...mfilocha:dev-1.17-pl.1-master - I'm going to split it into smaller PRs for review. 
But this should happen on a separate branch in the upstream repository, as soon as the localization team is established, I guess.

**Comment by sftim** [comment]
So it sounds to me that if an admin creates a new branch _dev-1.17-pl.1_ in **this** repo, @mfilocha you can send in pull requests that target that branch and that you're all ready to do that?

(and then those commits can merge into _master_ in a single PR that does preview OK)

Have I got that right?

**Comment by mfilocha** [comment]
@sftim Yes, I'm ready to send PR to that branch. I hope I understand it right that the PR to add localization team (https://github.com/kubernetes/org/pull/1508) can be approved regardless of the status of this PR, so that new team can approve PRs in the newly created branch.

**Comment by rlenferink** [comment]
To better follow the process on the localization page and be able to split up content to review in smaller PRs I have just created the *dev-1.17-pl.1* branch. Content to this branch can be added and reviewed in separate pull requests. When the [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content) is in place a pull request from *dev-1.17-pl.1* to *master* can be created which only needs sign-off because the actual content has already been reviewed at that point in time.

For this pull request I changed the base branch from *master* to *dev-1.17-pl.1* as well.

**Comment by mfilocha** [comment]
Above PRs contain required minimum translated content, split by smaller pieces.

File: README-pl.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Dokumentacja Kubernetes
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
# Dokumentacja projektu Kubernetes
```

File: README-pl.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Witamy!
+
+W tym repozytorium znajdziesz wszystko, czego potrzebujesz do zbudowania [strony internetowej Kubernetes wraz z dokumentacją](https://kubernetes.io/). Bardzo nam miło, że chcesz wziąć udział w jej współtworzeniu!
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
W tym repozytorium znajdziesz wszystko, czego potrzebujesz do zbudowania [strony internetowej Kubernetesa wraz z dokumentacją](https://kubernetes.io/). Bardzo nam miło, że chcesz wziąć udział w jej współtworzeniu!
```

File: README-pl.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## Twój wkład w dokumentację
+
+Możesz kliknąć w przycisk **Fork** w prawym górnym rogu ekranu, aby stworzyć kopię tego repozytorium na swoim koncie GitHub. Taki rodzaj kopii (odgałęzienia) nazywa się *fork*. Zmieniaj w nim, co chcesz, a kiedy będziesz już gotowy/a przesłać te zmiany do nas, przejdź do swojej kopii i stwórz nowy *pull request*, abyśmy zostali o tym poinformowani.
+
+Po stworzeniu *pull request*, jeden z recenzentów Kubernetes podejmie się zadania przekazania jasnych wskazówek pozwalających podjąć konkretne działania. Na Tobie, jako właścicielu *pull request*, **spoczywa odpowiedzialność za wprowadzenie poprawek zgodnie z uwagami recenzenta Kubernetes.** Może też się zdarzyć, że swoje uwagi zgłosi więcej niż jeden recenzent, lub że recenzję będzie robił ktoś inny, niż ten, kto został przydzielony na początku. W niektórych przypadkach, jeśli zajdzie taka potrzeba, recenzent może poprosić dodatkowo o recenzję jednego z [recenzentów technicznych Kubernetes](https://github.com/kubernetes/website/wiki/Tech-reviewers). Recenzenci zrobią wszystko, aby odpowiedzieć sprawnie, ale konkretny czas odpowiedzi zależy do różnych czynników.
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
Po stworzeniu *pull request*, jeden z recenzentów projektu Kubernetes podejmie się przekazania jasnych wskazówek pozwalających podjąć nastepne działania. Na Tobie, jako właścicielu *pull requesta*, **spoczywa odpowiedzialność za wprowadzenie poprawek zgodnie z uwagami recenzenta.** Może też się zdarzyć, że swoje uwagi zgłosi więcej niż jeden recenzent, lub że recenzję będzie robił ktoś inny, niż ten, kto został przydzielony na początku. W niektórych przypadkach, jeśli zajdzie taka potrzeba, recenzent może poprosić dodatkowo o recenzję jednego z [recenzentów technicznych](https://github.com/kubernetes/website/wiki/Tech-reviewers). Recenzenci zrobią wszystko, aby odpowiedzieć sprawnie, ale konkretny czas odpowiedzi zależy do wielu czynników.
```

File: README-pl.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+|[README po francusku](README-fr.md)|[README po koreańsku](README-ko.md)|
+|[README po niemiecku](README-de.md)|[README po portugalsku](README-pt.md)|
+|[README w hindi](README-hi.md)|[README po hiszpańsku](README-es.md)|
+|[README po indonezyjsku](README-id.md)|[README po chińsku](README-zh.md)|
+|[README po japońsku](README-ja.md)|[README po polsku](README-pl.md)|
+|||
+
+## Jak uruchomić serwis internetowy lokalnie przy pomocy Dockera?
+
+Zalecaną metodą uruchomienia serwisu internetowego Kubernetes lokalnie jest użycie specjalnego obrazu [Dockera](https://docker.com), który zawiera generator stron statycznych [Hugo](https://gohugo.io).
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
Zalecaną metodą uruchomienia serwisu internetowego Kubernetesa lokalnie jest użycie specjalnego obrazu [Dockera](https://docker.com), który zawiera generator stron statycznych [Hugo](https://gohugo.io).
```

File: README-pl.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+make docker-image
+```
+
+Po zbudowaniu obrazu, możesz uruchomić serwis lokalnie:
+
+```bash
+make docker-serve
+```
+
+Otwórz w przeglądarce adres http://localhost:1313, aby obejrzeć zawartość serwisu. Po każdej zmianie plików źródłowych, Hugo automatycznie aktualizuje stronę i odświeża jej widok w przeglądarce.
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
Aby obejrzeć zawartość serwisu otwórz w przeglądarce adres http://localhost:1313. Po każdej zmianie plików źródłowych, Hugo automatycznie aktualizuje stronę i odświeża jej widok w przeglądarce.
```

File: README-pl.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+```bash
+make serve
+```
+
+Zostanie uruchomiony lokalny serwer Hugo na porcie 1313. Otwórz w przeglądarce adres http://localhost:1313, aby obejrzeć zawartość serwisu. Po każdej zmianie plików źródłowych, Hugo automatycznie aktualizuje stronę i odświeża jej widok w przeglądarce.
+
+## Społeczność, listy dyskusyjne, uczestnictwo i wsparcie
+
+Zajrzyj na stronę [społeczności](http://kubernetes.io/community/), aby dowiedzieć się, jak możesz się zaangażować w jej działania.
```
Comment (kpucynski) [inline README-pl.md]:
```suggestion
Zajrzyj na stronę [społeczności](http://kubernetes.io/community/), aby dowiedzieć się, jak możesz zaangażować się w jej działania.
```

**Comment by sftim** [comment]
@nvtkaszpir can you add /lgtm to this PR if you're happy with it? You're listed as a future reviewer.

**Comment by rlenferink** [comment]
2 times `/lgtm` by pl-reviewers. I think this is good to be merged @sftim 🎉

**Comment by zacharysarah** [comment]
Congratulations! 🎉

**Pull request merged**